src/main/java/dev/adventurecraft/awakening/script/ScriptUI.java

```diff
@@ -48,10 +49,12 @@ public boolean getFancyGraphics() {
     }
 
     public boolean getHudEnabled() {
-        return ((ExInGameHud) this.mc.overlay).getHudEnabled();
+        return ((ExWorldProperties)this.mc.world.properties).getHudEnabled();
```

Comment:
I'm not too keen on this. The player can change `this.mc.overlay` and `getHudEnabled()` won't reflect it anymore.

Reply:
Don't worry. Just look into the MixinIngameHud class change.
I was not a fan of this, but because you can set ui.hudEnabled directly, there's not other choice (as far as I know).